### PR TITLE
[web] 130 algo store module

### PR DESCRIPTION
**Pull request by elliotmessi**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -77,3 +77,6 @@ FIFTYONE_HOST_URL=${FIFTYONE_HOST_IP}:${FIFTYONE_HOST_PORT}
 
 # viewer
 MONGODB_HOST_PORT=27019
+
+# algorithm store address, for front end
+ALGORITHM_STORE_URL=
```
Comment (phoenix-xhuang) [inline .env]: DEPLOY_MODULE_URL

Comment (elliotmessi) [reply .env]: fixed

File: .env [hunk 2]
```diff
@@ -77,3 +77,6 @@ FIFTYONE_HOST_URL=${FIFTYONE_HOST_IP}:${FIFTYONE_HOST_PORT}
 
 # viewer
 MONGODB_HOST_PORT=27019
+
+# algorithm store address, for front end
```
Comment (phoenix-xhuang) [inline .env]: deploy module address, for web integration.

Comment (elliotmessi) [reply .env]: fixed

**Review by phoenix-xhuang — APPROVED** [review]

**Review by IJtLJZ8Rm4Yr — APPROVED** [review]

**Pull request merged**